source/releasenotes/2024-04-xx-pantheon-mu-plugin-1-4-0-update.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Pantheon MU Plugin v1.4.0 update
+published_date: "2024-04-xx"
+categories: [wordpress, plugins]
+---
+
+With the latest [1.4.0 release](https://github.com/pantheon-systems/pantheon-mu-plugin/releases) of the Pantheon MU Plugin, we have updated the default cache TTL to one week. Previously, the default value was 10 minutes. This change is intended to improve the performance of your site by reducing the number of cache clears that are run automatically. The [Pantheon Advanced Page Cache](https://wordpress.org/plugins/pantheon-advanced-page-cache) plugin auto-clears content when pages are published or updated which is why we are increasing the default TTL. We highly recommend using the Pantheon Advanced Page Cache plugin to ensure that your site is performing optimally.
```

Comment:
```suggestion
With the latest [1.4.0 release](https://github.com/pantheon-systems/pantheon-mu-plugin/releases) of the Pantheon MU Plugin, we have updated the default cache max-age used to one week. Previously, the default value was 10 minutes. This change is intended to improve the performance by increasingly the likelihood that an unexpired cached copy of a page is present within the CDN. Often such increases in max-age settings carry the risk that site visitors see stale content that should be replaced by newly published posts. Pantheon greatly mitigates this risk with The [Pantheon Advanced Page Cache](https://wordpress.org/plugins/pantheon-advanced-page-cache) plugin. This plugin granularly clears cached responses using post IDs and other metadata. We highly recommend using the Pantheon Advanced Page Cache plugin to ensure that your site is performing optimally.
```

Reply:
We should say "max-age" and not "TTL" when the header value is literally "max-age".

Part of the reason I think this pedantry is valuable is that I have encountered a lot of customers who think this value is a the exact amount of time the response will be stored in the cache. "Time To Live" encourages that interpretation. It implies that this is the amount of time the response will live in the cache.

But that's not what it is. It is the _maximum_ amount of time the response should be kept. It is not a guarantee that the the response will be kept that long.

Reply:
The setting in the WordPress admin is "TTL", though. We can change that, but having two different phrases that mean the same thing would be just as confusing.

@scottbuscemi up to you whether we want to change the description in WordPress admin.

Reply:
> We can change that, but having two different phrases that mean the same thing would be just as confusing.

Yes, using both without explaining would be even more confusing.

After reading https://en.wikipedia.org/wiki/Time_to_live I am willing to drop this concern. Our usage does fit with with the meaning of TTL and I don't think it's worth the effort to change our terminology across the board. While I think people would be less likely to get confused if we used "max-age" everywhere instead of "TTL" there still would be some misinterpretation, changing UI copy creates swirl/confusion, etc

Reply:
We've got an internal ticket to change "ttl" to "max age" in the mu-plugin as part of the 1.4.0 update to the mu-plugin.